avatars/vrarmik/LegsManager.js

```diff
@@ -54,6 +54,8 @@ class Leg {
     this.foot.endTransform = new THREE.Object3D();
     this.foot.startHmdFloorTransform = new THREE.Object3D();
 
+    this.toe = new THREE.Object3D();
```

Comment:
Doesn't this toe need to be connected up to rotate correctly? See the lines below.